gcloud-Kms-decrypt: new page. (#11689)
tldr-pages/tldr: pages/common/gcloud-kms-decrypt.md

# gcloud kms decrypt

> > Decrypt a ciphertext file using a Cloud KMS key.
> > More information: <https://cloud.google.com/sdk/gcloud/reference/kms/decrypt>.

Reviewer: ```suggestion
> Decrypt a ciphertext file using a Cloud KMS key.
> See also: `gcloud`.
> More information: <https://cloud.google.com/sdk/gcloud/reference/kms/decrypt>.
```

- Decrypt a file using a specified key, key ring, and location:

> `gcloud kms decrypt --key=frodo --keyring=fellowship --location=global --ciphertext-file=path/to/input/ciphertext --plaintext-file=path/to/output/plaintext.dec`

Reviewer 2: ```suggestion
`gcloud kms decrypt --key={{key_name}} --keyring={{keyring_name}} --location={{global}} --ciphertext-file={{path/to/ciphertext}} --plaintext-file={{path/to/plaintext}}`
```

We usually put things that can be changed by the user into `{{}}`

> - Decrypt a file with additional authenticated data (AAD) and write the decrypted plaintext to stdout:
> 
> `gcloud kms decrypt --key=frodo --keyring=fellowship --location=global --additional-authenticated-data-file=path/to/aad --ciphertext-file=path/to/input/ciphertext --plaintext-file=-`

Reviewer 2: ```suggestion
- Decrypt a file with additional authenticated data (AAD) and write the decrypted plaintext to `stdout`:

`gcloud kms decrypt --key={{key_name}} --keyring={{keyring_name}} --location={{global}} --additional-authenticated-data-file={{path/to/file.aad}} --ciphertext-file={{path/to/ciphertext}} --plaintext-file=-`
```
